refaster-support/src/main/java/tech/picnic/errorprone/refaster/annotation/OnlineDocumentation.java

```diff
@@ -0,0 +1,49 @@
+package tech.picnic.errorprone.refaster.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Signals that a Refaster rule or group of Refaster rule comes with online documentation.
```

Comment:
```suggestion
 * Signals that a Refaster rule or group of Refaster rules comes with online documentation.
```

Reply:
That said, there are more references to be updated in this PR (and even in this file :eyes:).